static/js/compose_height.js

```diff
@@ -0,0 +1,52 @@
+import autosize from "autosize";
+import $ from "jquery";
+
+let is_full_size = false; // true or false
+
+function get_ele_height(selector) {
+    return Number.parseInt($(selector).css("height"), 10);
+}
+
+function get_expected_full_height() {
+    // This returns the full size of the compose box
+    // Panels can have different height and this calculates it accordignly
+    return window.innerHeight - 100 - get_ele_height("#message_view_header") - get_ele_height("#panels");
+}
+
+// Some functions to handle the state explicitely
+export function set_is_full_size(is_full) {
+    is_full_size = is_full;
+}
+
+export function get_is_full_size() {
+    return is_full_size;
+}
```

Comment:
I think `compose_ui.is_full_size()` and `compose_ui.set_full_size(value)` would be better names for these functions, assuming we put them in that module.

Reply:
Sure, updated.

Reply:
Sorry, to be clear, my proposal was to put this code in `compose_ui.js`, not have a new module.

Reply:
Sure!